### PR TITLE
docs: homepage tweaks

### PR DESCRIPTION
**Pull request by wodeni**
# Description

This PR makes a few changes to our homepage:

* Add discord server invitation
* Tweak landing page design
  * Better `Demo` component positioning and sizing
  * More demo examples: hexagonal graph, caffeine, 3D euler diagram
* Update footer to include copyright and license info   
* Gallery page: rounded corners and dark-mode shadow

# Checklist

- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have reviewed any generated registry diagram changes

### CODE REVIEW

File: packages/docs-site/.vitepress/config.ts [hunk 1]
```diff
@@ -163,6 +163,7 @@ export default defineConfig({
       },
       { text: "Documentation", link: "/docs/ref", activeMatch: "/docs/ref" },
       { text: "Try Penrose", link: "pathname:///try/index.html" },
+      { text: "Join Discord", link: "https://discord.gg/Y3K2kHxp2b" },
```
Comment (samestep) [inline packages/docs-site/.vitepress/config.ts:166]: Did you configure this to be a permanent link? (The default expiration time is one week.)

File: packages/docs-site/index.md [hunk 2]
```diff
@@ -3,24 +3,22 @@ layout: home
...
       text: Try
       link: pathname:///try/index.html
     - theme: alt
-      text: Video
-      link: https://vimeo.com/416822487
+      text: Tutorial
+      link: /docs/tutorial/welcome
 features:
   - title: Declarative
     details: Penrose is a platform that enables people to create beautiful diagrams just by typing mathematical notation in plain text.
```
Comment (samestep) [inline packages/docs-site/index.md:19]:
```suggestion
    details: Penrose is a platform that enables people to create beautiful diagrams just by typing notation in plain text.
```

File: packages/docs-site/.vitepress/config.ts [hunk 3]
```diff
@@ -286,6 +288,10 @@ export default defineConfig({
       ],
     },
 
-    footer: { copyright: "made with ❤️ in Pittsburgh and abroad" },
+    footer: {
+      message:
+        'Released under the <a href="https://github.com/penrose/penrose/blob/main/LICENSE">MIT License</a>.',
```
Comment (samestep) [inline packages/docs-site/.vitepress/config.ts:293]: This link works but doesn't look like a link on the page. Is there a way to style it to look clickable?

File: packages/editor/src/components/ExamplesBrowser.tsx [hunk 4]
```diff
@@ -27,6 +24,8 @@ const Example = ({
   loadExample: (t: TrioWithPreview) => Promise<void>;
   k: number;
 }) => {
+  const parser = new DOMParser();
+  const serializer = new XMLSerializer();
```
Comment (samestep) [inline packages/editor/src/components/ExamplesBrowser.tsx:28]: Is this just to reduce build warnings?

**Review by samestep — COMMENTED** [review]
Looks great, thanks for doing this! Just had a few comments.

File: packages/docs-site/.vitepress/config.ts (hunk 1 above)
Comment (wodeni) [reply packages/docs-site/.vitepress/config.ts:166]: Yep

File: packages/docs-site/.vitepress/config.ts (hunk 3 above)
Comment (wodeni) [reply packages/docs-site/.vitepress/config.ts:293]: Not sure. I only followed the vitepress guide here: https://vitepress.dev/reference/default-theme-footer#footer for this one. Probably not high priority

**Pull request merged**